packages/client/package.json

```diff
@@ -22,17 +22,13 @@
     "src",
     "README.md"
   ],
-  "source": "./src/index.ts",
```

Comment:
Should we also remove `src` from `files` above since we don't export sources anymore?

Reply:
Done!